### ISSUE TITLE
Error while publishing snupkg package

### ISSUE THREAD

**Issue by raczeja** [NuGet/NuGetGallery#7949]
### Describe the bug
I'm receiving following email while publishing snupkg package
[NuGet Gallery] Symbol package validation failed - OcarambaLite 3.2.9
The symbol package OcarambaLite 3.2.9 failed validation because of the following reason(s):
•	The uploaded symbols package contains pdb(s) for a corresponding dll(s) not found in the nuget package.
Your symbol package was not published on NuGet Gallery and is not available for consumption.
You can reupload your symbol package once you've fixed the issue with it.

link to csproj file https://github.com/ObjectivityLtd/Ocaramba/blob/master/OcarambaLite/OcarambaLite.csproj 

nuget package of version 3.2.9 was published https://www.nuget.org/packages/OcarambaLite/

 I set:
IncludeSymbols>true
SymbolPackageFormat>snupkg
DebugType>portable

### To Reproduce
1. Build solution 
2. Try to publish snupkg file https://ci.appveyor.com/project/ObjectivityAdminsTeam/ocaramba/builds/32196042/job/vm9egajvaas90r9k/artifacts 
3. See error

### Expected behavior
snupkg should be published with success

**Comment by loic-sharma** [comment]
cc @ryuyu who is handling support this week.

**Comment by raczeja** [comment]
is it possible that it could be cause by targeting also Net 4.5 ?
`
<TargetFrameworks>netcoreapp3.1</TargetFrameworks>
`
`
<TargetFrameworks Condition="'$(OS)' != 'Unix'">net45;net472;netcoreapp3.1</TargetFrameworks>
`
but according to [PDB formats](https://devblogs.microsoft.com/nuget/improved-package-debugging-experience-with-the-nuget-org-symbol-server/)

> Q. Which PDB formats are supported?
Currently, NuGet.org symbol server supports only the portable PDB format. The portable PDB format can be created and consumed for any .NET application provided that the consuming client is on .NET 4.7.1 or later. More info about Portable PDBs is available on the .NET GitHub.

**Comment by joelverhagen** [comment]
I looked at the PDBs and DLLs that you submitted in the .snupkg and .nupkg, respectively.

TFM | DLL Signature | PDB Signature
--- | --- | ---
net45 | 6b64eae4-92c0-4aba-943f-4c0ab3d794e0 | 004f0c75-eb0d-4174-8b80-ddc6ac33179c
net472 | 84a9ec41-a08f-4a1a-a2dc-7cb3ea854810 | 15016f14-5b1d-4ce8-93eb-dac0c5398287
netcoreapp3.1 | 29900157-0ea0-42a5-b9ca-b8c3d1a3c144 | 5181a5f4-adde-4c56-b90b-10ba898ae1fe

It looks like the PDBs were perhaps built at a different time or from slightly different input than the DLL. Could you check your build process to ensure the .nupkg and .snupkg were built and packed at the same time?

**Comment by joelverhagen** [comment]
> is it possible that it could be cause by targeting also Net 4.5 ?

As far as I can tell, all three of the PDBs were portable PDBs (which is what is accepted on nuget.org). I think the problem is the mismatched signature.

BTW, I used JetBrains dotPeek to look at the signature in the DLL and PDB.

**Comment by raczeja** [comment]
1. First time I generated .nupkg and .snupkg with <DebugType>full</DebugType>
2. I got error message while publishing .snupkg 
3. I changed  <DebugType>portable</DebugType> and regenerated .nupkg and .snupkg for the same version and I was trying to publish only .snupkg , that's why the Signature are different
4. I generated new version .nupkg and .snupkg (without <DebugType> at all) and published both without problems

thank you for your help
